miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppDownloader.kt

```diff
@@ -56,6 +56,7 @@ internal class MiniAppDownloader(
                     storage.saveFile(file, baseSavePath, response.byteStream())
                 }
                 miniAppStatus.setVersionDownloaded(appId, versionId, true)
+                storage.removeOutdatedVersionApp(appId, versionId)
```

Comment:
The display doesn't need to be blocked/waiting for deletion. This should happen in parallel. What do you think?

Reply:
Good point. Should run in parallel for deletion because it not affects the main flow usage.